runtime/src/main/resources/application-plugin-flexibleengine.properties

```diff
@@ -0,0 +1,6 @@
+#
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Huawei Inc.
+#
+#
+plugins.enabled=flexibleengine
```

Comment:
@baixinsui it is better if we can avoid these new profiles. Instead get available CSPs directly from the plugin manager.

Reply:
@swaroopar Without these configuration files, we can only decide which plugins are supported when packaging the executable Jar with flag '-Pplugin-xxx'.

Reply:
According to the discussion results, I have deleted these configurations and related codes.